plugins/lint/src/interfaces.ts

```diff
@@ -0,0 +1,27 @@
+import stylelint from 'stylelint';
+
+export interface LintArgs {
+    /** Attempt to fix the lint errors. */
+    fix?: boolean;
+    /** Do not use any cached results from previous runs. */
+    noCache?: boolean;
+    /** Post lint results as annotations to PR. Only in CI. */
+    annotate?: boolean;
+    /** An optional list of files to lint */
+    files?: string[];
+}
+
+export type StylelintResult = stylelint.LinterResult & {
```

Comment:
since this is only used for stylelint it think we should move it to that file. In general I like to keep interfaces where they are used